### ISSUE TITLE
Bug(?): Delta-V won't show when fuel tanks aren't attached to a capsule.

### ISSUE THREAD

**Issue by coldbluelava** [CYBUTEK/KerbalEngineer#111]
Better displayed with images: 
https://imgur.com/a/OVIRi

**Comment by Gerry1135** [comment]
Your images don't show any obvious issues.  There is no deltaV in picture 2 because there is no engine.  There is none in picture 3 because the first stage is decoupling the fuel tank and engine from the pod.  The same is true in picture 4.

Basically, you need to keep an eye on the staging setup because KSP will often set things up in a ridiculous way when you attach and detach things.

**Comment by Benargee** [comment]
Yeah, no bug. You're decoupling your engine and fuel before you have a chance to use them.
